docs/core/tools/sdk-errors/index.md

```diff
@@ -0,0 +1,186 @@
+|NETSDK1023|A PackageReference for '{0}' was included in your project. This package is implicitly referenced by the .NET SDK and you do not typically need to reference it from your project. For more information, see {1}|
+|NETSDK1024|Folder '{0}' already exists either delete it or provide a different ComposeWorkingDir|
+|NETSDK1025|The target manifest {0} provided is of not the correct format|
+|NETSDK1028|Specify a RuntimeIdentifier|
+|NETSDK1029|Unable to use '{0}' as application host executable as it does not contain the expected placeholder byte sequence '{1}' that would mark where the application name would be written.|
+|NETSDK1030|Given file name '{0}' is longer than 1024 bytes|
+|NETSDK1031|It is not supported to build or publish a self-contained application without specifying a RuntimeIdentifier. You must either specify a RuntimeIdentifier or set SelfContained to false.|
+|NETSDK1032|The RuntimeIdentifier platform '{0}' and the PlatformTarget '{1}' must be compatible.|
+|NETSDK1042|Could not load PlatformManifest from '{0}' because it did not exist.|
+|NETSDK1043|Error parsing PlatformManifest from '{0}' line {1}.  Lines must have the format {2}.|
+|NETSDK1044|Error parsing PlatformManifest from '{0}' line {1}.  {2} '{3}' was invalid.|
+|[NETSDK1045](netsdk1045.md)|The current .NET SDK does not support targeting {0} {1}.  Either target {0} {2} or lower, or use a version of the .NET SDK that supports {0} {1}.|
+|NETSDK1046|The TargetFramework value '{0}' is not valid. To multi-target, use the 'TargetFrameworks' property instead.|
+|NETSDK1047|Assets file '{0}' doesn't have a target for '{1}'. Ensure that restore has run and that you have included '{2}' in the TargetFrameworks for your project. You may also need to include '{3}' in your project's RuntimeIdentifiers.|
+|NETSDK1048|'AdditionalProbingPaths' were specified for GenerateRuntimeConfigurationFiles, but are being skipped because 'RuntimeConfigDevPath' is empty.|
+|NETSDK1049|Resolved file has a bad image, no metadata, or is otherwise inaccessible. {0} {1}|
+|NETSDK1050|The version of Microsoft.NET.Sdk used by this project is insufficient to support references to libraries targeting .NET Standard 1.5 or higher.  Please install version 2.0 or higher of the .NET Core SDK.|
+|NETSDK1051|Error parsing FrameworkList from '{0}'.  {1} '{2}' was invalid.|
+|NETSDK1052|Framework list file path '{0}' is not rooted. Only full paths are supported.|
+|NETSDK1053|Pack as tool does not support self contained.|
+|NETSDK1054|only supports .NET Core.|
+|NETSDK1055|DotnetTool does not support target framework lower than netcoreapp2.1.|
+|NETSDK1056|Project is targeting runtime '{0}' but did not resolve any runtime-specific packages. This runtime may not be supported by the target framework.|
+|NETSDK1057|You are using a preview version of .NET. See: aka.ms/dotnet-support-policy|
```

Comment:
```suggestion
|NETSDK1057|You are using a preview version of .NET. See <https://aka.ms/dotnet-support-policy>.|
```

Reply:
I had removed https:// because markdownlint complained about bare URLs.  I'll try this and see if it elicits the same response.